### ISSUE TITLE
Low access improvements

### ISSUE THREAD

**Issue by Piras314** [Goob-Station/Goob-Station#623]
make a lower access that just lets you go in hallways and stuff like that for every department, like what security has with the "brig" access. this also wouldn't just apply to doors but also like machines, like the anomaly generator for example shouldn't be under science access

**Comment by HankKitts** [comment]
I don't agree completely, I think access should be only for low pop times.
Players stealing/breaking things could be a major issue with this and it'd also remove the point of stealing IDs.

**Comment by Piras314** [comment]
nah
